ccm_web/server/src/api/api.utils.ts

```diff
@@ -37,19 +41,32 @@ export function parseErrorBody (body: unknown): string {
   if (!isCanvasErrorBody(body)) {
     return `Response body had unexpected shape: ${JSON.stringify(body)}`
   }
-  return body.errors.map(e => e.message).join(' ')
+  let errorMessage: string
+  try {
+    errorMessage = body.errors.map(e => e.message).join(' ')
+  } catch (e) {
+    errorMessage = JSON.stringify(body)
+    logger.debug(errorMessage)
+  }
+  return errorMessage
```

Comment:
Just as an aside, I believe you made these changes because of runtime errors caused by changes to the `isCanvasErrorBody` type guard. Hopefully we can resolve that or push off those changes so this can be removed.

Reply:
See my PR https://github.com/lsloan/ccm/pull/7